lib/filters/bucket/pagecontent.js

```diff
@@ -264,7 +264,7 @@ PCBucket.prototype.getFormatRevision = function(restbase, req) {
                 .then(function(apiRes) {
                     if (apiRes.status === 200) {
                         var apiRev = apiRes.body.items[0].revisions[0];
-                        var tid = rbUtil.tidFromDate(new Date(apiRev.timestamp));
+                        var tid = rbUtil.tidFromDate(+apiRev.timestamp);
```

Comment:
Coerces to numeric, just to be safe.  Should catch the oddball case where (say) date.getTime() is a method, but returns some non-numeric value, like `{}`, since `isNaN(+{}) === true`.